elements/pfe-jump-links/demo/index.html

```diff
@@ -57,36 +56,40 @@ <h1><code>pfe-jump-links</code></h1>
       position:sticky;
       top: 0;
     }
+    body {
+      font-family: 'Red Hat Text', sans-serif;
+      font-weight: 400;
+    }
   </style>
       
   <h2 style="text-align: center; margin: 60px auto;">PFE Jump Links - standard</h2>
   <div style="margin-bottom: 400px;" class="pfe-l-grid">
     <div class="sticky pfe-l-grid__item pfe-m-12-col pfe-m-3-col-on-lg">
       <pfe-jump-links-nav id="jumplinks1" default>
-        <h4 class="heading" slot="heading">Jump to section</h4>
+        <h4 class="pfe-jump-links-nav--heading" slot="pfe-jump-links-nav--heading">Jump to section</h4>
```

Comment:
```suggestion
        <h4 class="pfe-jump-links__heading" slot="pfe-jump-links__heading">Jump to section</h4>
```

Regions are underscore prefixed, with modifiers and properties being dash separated.

Reply:
Oh these classes are on the demo page.  What do the classes in the light DOM provide that the component is not offering?